includes/tax-functions.php

```diff
@@ -442,3 +410,63 @@ function edd_download_is_tax_exclusive( $download_id = 0 ) {
 
 	return (Bool) apply_filters( 'edd_download_is_tax_exclusive', $ret, $download_id );
 }
+
+/**
+ * Gets the tax rate object from the database.
+ * Used in `edd_get_tax_rate`, `edd_build_order`, `edd_add_manual_order`.
+ *
+ * @param array $args
+ * @return object|boolean
+ *
+ * @since 3.0
+ */
+function edd_get_tax_rate_object( $args ) {
```

Comment:
I wonder if we should make this more specific. It's really _"get tax rate object for given country / region"_, yes? So although it's wordy, maybe:

`edd_get_tax_rate_object_for_country_region()`
or
`edd_get_tax_rate_object_for_location()`
or
`edd_get_tax_rate_object_for_address()` (might be misleading since we don't need the whole address)

Reply:
I think location makes sense, while still being somewhat generic. I had originally used `object` in the name because `edd_get_tax_rate` is already in use...another option might be `edd_get_location_tax_rate`, which would remove the `for`, and I don't think the code base usually includes `object` or similar in function names?

Reply:
I think I prefer `edd_get_tax_rate_for_location()`. I know the `for` feels silly, but I like how the item I'm retrieving is closer to the start of the name. To me it just reads better/faster while skimming. I quickly know "I'm getting a tax rate".